src/install-peerdeps.js

```diff
@@ -144,12 +147,40 @@ function getPackageJson(
   });
 }
 
+/**
+ * Builds the package install string based on the version
+ * @param {Object} options - information needed to build a package install string
+ * @param {string} optoins.name - name of the package
+ * @param {string} options.version - version string of the package
+ * @returns {string} - the package name and version formatted for an install command
+ */
+const getPackageString = ({ name, version }) => {
+  // check for whitespace
+  if (version.indexOf(" ") >= 0) {
```

Comment:
is there a reason to branch here, instead of using `semver` logic unconditionally?